### PR TITLE
Cerenss/issue6

### PR DESCRIPTION
**Pull request by cerenss**
Added the **"My Learning"** section to the **Home Page.** 

The section currently uses **mock data from the data folder**, as the API data is not yet implemented. 

This commit introduces the **initial structure, layout, and styling** for the "My Learning" section, **ready for integration with the API in future updates.**

### CODE REVIEW

**Comment by cerenss** [comment]
There is nothing displayed in the Netlify preview. I don't know what is the problem.

**Comment by wasimtayyan** [comment]
> There is nothing displayed in the Netlify preview. I don't know what is the problem.
> 
> ** I need to navigate to _/main/home_ manually. Is it how it supposed to be, or there is something I am missing?

> No there is nothing wrong with manually navigating for now because we will navigate the pages later on by navbar

File: app/page.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
+'use client';
 import Image from 'next/image'
+import React, { useState } from 'react'
+import Page from './main/home/page'
+
```
Comment (wasimtayyan) [inline app/page.js:5]: you don't need 'use client' & these imports here

File: package.json [hunk 2]
```diff
@@ -9,13 +9,20 @@
...
+    "@mui/icons-material": "^5.11.16",
+    "@mui/material": "^5.13.6",
     "autoprefixer": "10.4.14",
     "eslint": "8.43.0",
     "eslint-config-next": "13.4.7",
     "next": "13.4.7",
     "postcss": "8.4.24",
     "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "tailwindcss": "3.3.2"
```
Comment (wasimtayyan) [inline package.json:19]: why you deleted those ?!

File: data/index.html
Comment (wasimtayyan) [inline data/index.html:1]: Are you need this file for the mock API ??

**Review by wasimtayyan — CHANGES_REQUESTED** [review]
easy to read code because of good comments but there is a missing dependencies from [package.json] file why !

File: package.json (hunk 2 above)
Comment (cerenss) [reply package.json:19]: I did not delete any of these by hand, there could be some mistake

File: data/index.html
Comment (cerenss) [reply data/index.html:1]: I was checking something, sorry forgot it was there. I will delete it

File: app/page.js (hunk 1 above)
Comment (cerenss) [reply app/page.js:5]: Will remove them

**Comment by vuslatsena** [comment]
It's a clean, well structured code 💕 I agree with Wasim's comments. After editing them, we will check again.

**Review by wasimtayyan — APPROVED** [review]

**Review by vuslatsena — APPROVED** [review]

File: app/main/home/page.jsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
-import React from "react";
+
 import FeaturedCourse from "../../../components/homePage-components/FeaturedCourse";
+import MyLearning from "../../../components/home-page-components/MyLearning";
 
 function page() {
-  return (
-    <div className="container mx-auto px-4  max-w-screen-xl">
```
Comment (wasimtayyan) [inline app/main/home/page.jsx:6]: use this class name for styling 
 and make "My Learning"  the same size as "featured courses" You can use this class name: text-3xl font-bold

**Review by wasimtayyan — COMMENTED** [review]
just some styling changes😅

**Pull request merged**